tests/test_download.py

```diff
@@ -7,3 +14,60 @@ def test_imports(self):
         )
 
         assert True
+
+    def test_common_crawl_urls(self):
```

Comment:
I think these tests are very good. I see we're not testing every `raise` that was added by this PR; I don't think that's an issue but just wanted to make sure we don't want them (`raise ValueError("No urls were provided to download")`, etc.).

Reply:
Good callout. I've added a few more tests.